app/assets/javascripts/miq_dynatree.js

```diff
@@ -321,6 +321,15 @@ function cfmeOnClick_GenealogyTree(id) {
   }
 }
 
+// OnCheck handler for the SmartProxy Affinity tree
+function cfmeOnClick_SmartProxyAffinityCheck(node) {
```

Comment:
@h-kataria since this is upstream, should this be prefixed with something other than `cfme`?